### PR TITLE
pi: allow incrementing ICMP id for each packet

### PR DESCRIPTION
**Pull request by lebauce**

### CODE REVIEW

**Comment by lebauce** [comment]
run skydive-functional-tests-backend-elasticsearch

File: packet_injector/client.go [hunk 1]
```diff
@@ -281,13 +287,13 @@ func (pc *PacketInjectorClient) onAPIWatcherEvent(action string, id string, reso
...
 
-//Start the PI client
+// Start the packet injector client
 func (pc *PacketInjectorClient) Start() {
 	pc.MasterElector.StartAndWait()
 	pc.watcher = pc.piHandler.AsyncWatch(pc.onAPIWatcherEvent)
 }
 
-//Stop the PI client
+// Stop the packet inector client
```
Comment (nplanel) [inline packet_injector/client.go]: injector

File: packet_injector/client.go [hunk 2]
```diff
@@ -247,17 +253,17 @@ func (pc *PacketInjectorClient) OnSwitchToSlave() {
 
 func (pc *PacketInjectorClient) onAPIWatcherEvent(action string, id string, resource types.Resource) {
 	logging.GetLogger().Debugf("New watcher event %s for %s", action, id)
-	ppr := resource.(*types.PacketParamsReq)
+	ppr := resource.(*types.PacketInjection)
```
Comment (masco-enovance) [inline packet_injector/client.go]: nit: how about some other name instead of ppr?

File: packet_injector/client.go [hunk 3]
```diff
@@ -296,7 +302,7 @@ func (pc *PacketInjectorClient) Stop() {
 func (pc *PacketInjectorClient) setTimeouts() {
 	injections := pc.piHandler.Index()
 	for _, v := range injections {
-		ppr := v.(*types.PacketParamsReq)
+		ppr := v.(*types.PacketInjection)
```
Comment (masco-enovance) [inline packet_injector/client.go]: same

File: packet_injector/inject_packet.go [hunk 4]
```diff
@@ -237,6 +245,13 @@ func InjectPacket(pp *PacketParams, g *graph.Graph, chnl *Channels) (string, err
 				if i != pp.Count-1 {
 					time.Sleep(time.Millisecond * time.Duration(pp.Interval))
 				}
+
+				if pp.Increment {
```
Comment (masco-enovance) [inline packet_injector/inject_packet.go]: call only for ICMP packets

File: packet_injector/client.go (hunk 2 above)
Comment (lebauce) [reply packet_injector/client.go]: done

File: packet_injector/client.go (hunk 3 above)
Comment (lebauce) [reply packet_injector/client.go]: done

File: packet_injector/inject_packet.go (hunk 4 above)
Comment (lebauce) [reply packet_injector/inject_packet.go]: fixed

**Pull request merged**